assets/environment-interpreter/wrapper/wrapper

```diff
@@ -8,6 +8,11 @@
 export _flox_activate_tracelevel="${_FLOX_PKGDB_VERBOSITY:-0}"
 [ "$_flox_activate_tracelevel" -eq 0 ] || set -x
 
+function wrapperDebug {
+  [ -n "${FLOX_ACTIVATE_TRACE-}" ] || return 0
+  echo "[DEBUG] $*" >&2
+}
+
```

Comment:
Can drop this function, it was just helpful for this PR. That said, it is useful to have such utility features - we had a general-purpose debug function in the past for just this reason.

Reply:
This will be deleted as we create the `env --keep` equivalent script as described in other comment.

Reply:
It was in fact deleted. Resolving conversation.